### PR TITLE
Implement Affinity switch for `conduct run` task

### PR DESCRIPTION
**Pull request by fsat**
Restrict the `--affinity` switch for version 1.1 onwards

### CODE REVIEW

File: src/main/scala/com/typesafe/conductr/client/ConductRController.scala [hunk 1]
```diff
@@ -276,7 +276,8 @@ class ConductRController(conductr: Uri, loggingQuery: Uri, connectTimeout: Timeo
   }
 
   private def runBundle(runBundle: RunBundle): Unit = {
-    val uri = s"${apiVersionPath(runBundle.apiVersion)}/bundles/${runBundle.bundleId}?scale=${runBundle.scale}"
+    val affinityParam = runBundle.affinity.fold("")(otherBundleId => s"&affinity=$otherBundleId")
```
Comment (huntc) [inline src/main/scala/com/typesafe/conductr/client/ConductRController.scala]:
```scala
val affinityParam = runBundle.affinity.fold("")("&affinity=" + _)
```

**Comment by huntc** [comment]
LGTM. One small comment and perhaps squash the commits...

**Comment by fsat** [comment]
Manual test successfully done against ConductR master branch

**Comment by markusjura** [comment]
Shouldn't we add the logic that the `--affinity` option is only used onwards ConductR 1.1.x. For ConductR 1.0.x this option should ignored and the user should receive a warning if this option has been specified.

**Pull request merged**